code/_globalvars/lists/mobs.dm

```diff
@@ -28,6 +28,7 @@ GLOBAL_LIST_EMPTY(alive_living_list)		//all alive /mob/living, including clientl
 GLOBAL_LIST_EMPTY(carbon_list)				//all instances of /mob/living/carbon and subtypes, notably does not contain brains or simple animals
 GLOBAL_LIST_EMPTY(offered_mob_list)				//all /mobs offered by admins
 GLOBAL_LIST_EMPTY(ai_list)
+GLOBAL_LIST_EMPTY(hivemind_list)
```

Comment:
This doesn't seem to be used anymore.

Reply:
I use it to keep track of the hivemind in its Init and Destroy procs

Reply:
Yea, but what for?

Reply:
We track over mobs, it only makes sense no?

Reply:
If we are not using the list anywhere, then it's legacy that should be removed, really.
We already track xenos through the hive datum.